test/functional/tests/security/fuzzy/kernel/fuzzy_io_class/test_fuzzy_io_class_list_cache_id.py

```diff
@@ -0,0 +1,68 @@
+import pytest
+
+from api.cas import casadm
+from api.cas.cache_config import CacheMode, CacheLineSize, CleaningPolicy, UnalignedIo, \
+    KernelParameters
+from api.cas.casadm_params import OutputFormat
+from api.cas.cli import list_io_classes_cmd
+from core.test_run import TestRun
+from storage_devices.disk import DiskType, DiskTypeSet, DiskTypeLowerThan
+from test_tools.peach_fuzzer.peach_fuzzer import PeachFuzzer
+from tests.security.fuzzy.kernel.common.common import get_fuzz_config, validate_cas_working, \
+    create_files_on_open_cas
+
+mount_point = "/mnt/test"
+iterations_count = 1000
+
+
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.optane, DiskType.nand]))
+@pytest.mark.require_disk("core", DiskTypeLowerThan("cache"))
+@pytest.mark.parametrizex("cache_mode", CacheMode)
+@pytest.mark.parametrizex("cache_line_size", CacheLineSize)
+@pytest.mark.parametrizex("cleaning_policy", CleaningPolicy)
+@pytest.mark.parametrizex("unaligned_io", UnalignedIo)
+def test_fuzzy_io_class_list_cache_id(cache_mode, cache_line_size, cleaning_policy, unaligned_io):
+    """
+        title: Fuzzy test for casadm list IO class command – cache id
+        description: Using Peach Fuzzer check Open CAS ability of handling wrong cache id in
+            ‘list IO class’ command.
+        pass_criteria:
+            - System did not crash
+            - Open CAS still works.
+    """
+    with TestRun.step("Start cache and add core device, make filesystem and run fio"):
+        cache_disk = TestRun.disks['cache']
+        core_disk = TestRun.disks['core']
+        cache, core = validate_cas_working(core_disk, cache_disk, mount_point, cache_mode,
+                                           cache_line_size, KernelParameters(unaligned_io),
+                                           cleaning_policy)
+
+    with TestRun.step("Load default IO class configuration file"):
+        casadm.load_io_classes(cache.cache_id, '/etc/opencas/ioclass-config.csv')
+
+    with TestRun.step("Prepare PeachFuzzer"):
+        valid_value = str(core.cache_id).encode('ascii')
+        PeachFuzzer.generate_config(get_fuzz_config("cache_id.yml"))
+        base_cmd = list_io_classes_cmd("{param}", OutputFormat.table.name).encode('ascii')
+        commands = PeachFuzzer.get_fuzzed_command(base_cmd, iterations_count)
+
+    for index, cmd in TestRun.iteration(enumerate(commands), f"Run command {iterations_count} "
+                                                             f"times"):
+        with TestRun.step(f"Iteration {index + 1}"):
+            TestRun.LOGGER.info(f"Cache_id: {cmd.param}")
+            TestRun.LOGGER.info(f"Encoded command: {cmd.command}")
+            output = TestRun.executor.run(cmd.command)
+            if output.exit_code == 0 and \
+                    "Try `casadm --help | -H' for more information" not in output.stdout and \
+                    not len(output.stderr):
+                TestRun.LOGGER.info("List io classes command run successfully")
```

Comment:
Since this log doesn't contain any information it could be removed.  This change should be applied to all the tests.